web/controllers/game_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Yggdrasil.GameController do
+  use Yggdrasil.Web, :controller
+
+  alias Yggdrasil.Game
+
+  def index(conn, _params) do
+    games = Repo.all Game
```

Comment:
Ah no the goal isn't a repo but a game engine Api. This branch hasn't been updated yet but we split apps. 

I'd rather not let the game engine internsls bleed out of the Yggdrasil app so why I'm suggesting we introduce an api. 

So my goal is to let the game engine manage the life cycle of its core assets like character. I'm not sure the web needs to. 

So while I said Yggdrasil.Game the intent is a module that lets you interact with the game engine. 

So for example in my examples of chracater any modification or removing probably should be checked to ensure the user isn't running. Also if there are any other things like creating a character internally updates several tables to place in a default room etc the game engine is best suited for this not the web. 

So add_character in my example above maybe a direct mapping temporarily it will likely expand. Thus we can do that and not modify the web. 

All that said internal to the game engine if you need a character im all for using a repo and totally agree that that is best. Not suggesting this for that purely between apps.

Naming wise I'm good with what you suggested.